dpctl-capi/CMakeLists.txt

```diff
@@ -9,10 +9,14 @@ project(
 set(CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH} "${CMAKE_SOURCE_DIR}/cmake/modules/")
 find_package(Git REQUIRED)
 
-option(DPCTL_CUSTOM_DPCPP_INSTALL_DIR
+option(DPCTL_DPCPP_HOME_DIR
     "Use a custom version of DPCPP installed at the provided location."
```

Comment:
The message should say "The installation home for the DPCPP compiler."

Reply:
Good catch. Fixed.